upsertGitHubTag-f79c4f36-3e61-43f4-8f6c-0b2e0e9774d7/deployment/index.js

```diff
@@ -102,21 +98,23 @@ function processEvent(event, callback) {
     console.log(JSON.stringify(event));
     var loneEvent = event.Records[0];
     var requestBody = JSON.parse(loneEvent.body);
-    if (! verifyGitHub(requestBody)) {
+
+    // The payload is URI-encoded and encoded in base64
+    const uriDecoded = decodeURIComponent(requestBody.payload);
+    const buff = Buffer.from(uriDecoded, 'base64');
+    const body = JSON.parse(buff.toString('utf8'));
+
+    console.log('GitHub Payload');
```

Comment:
thought: 
if we only really look at this when there is an unexpected problem, maybe this should be nested under the `if` on 110